pkgs/top-level/all-packages.nix

```diff
@@ -12393,9 +12403,6 @@ in
     inherit (llvmPackages_11) llvm clang-unwrapped;
   };
 
-  libcxx = llvmPackages.libcxx;
-  libcxxabi = llvmPackages.libcxxabi;
```

Comment:
Those should be moved to aliases.nix

Reply:
Yes, this is already the case (/ purpose of this PR).